bindings/2.2074/GeometryDash.bro

```diff
@@ -8849,6 +8864,7 @@ class GameToolbox {
     static cocos2d::CCPoint getRelativeOffset(GameObject*, cocos2d::CCPoint) = win 0x64970, m1 0x43f1f4, imac 0x4dc100;
     static gd::string getResponse(cocos2d::extension::CCHttpResponse*) = win 0x64310, imac 0x4dba00, m1 0x43eb40;
     static gd::string getTimeString(int, bool) = win 0x65e20, imac 0x4de620, m1 0x44145c, ios 0x49338;
+    // on windows, 2nd param is ignored and assumed to be "a"
```

Comment:
personally i would move this to be on the same line as the function this comment is referring to. yknow, to keep things organized

Reply:
ah yeah, I did this because it was done at https://github.com/geode-sdk/bindings/blob/b6c1fa9a621b178deb721dff4209d810b5618995/bindings/2.2074/GeometryDash.bro#L7572

Reply:
done